src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy

```diff
@@ -115,9 +116,11 @@ public class ArtifactoryPermissionsUpdater {
         }
 
         if (apiOutputDir.exists()) {
-            throw new IOException(apiOutputDir.path + " already exists")
+            println "$apiOutputDir.path already exists" // not fatal
```

Comment:
This intends to ensure the correct removal of permissions for components whose files were removed since the previous execution. Otherwise those might be retained indefinitely if the environment (workspace deletion) is not properly set up.

Reply:
OK, can revert. This just made it a lot more convenient to iteratively test.